src/eduid_userdb/tests/test_profile.py

```diff
@@ -31,7 +31,7 @@ def test_create_profile(self):
     def test_create_profile_from_dict(self):
         data = dict(owner='test owner', created_by='test created_by', created_ts=True, schema='test schema')
         data['profile_data'] = OPAQUE_DATA
-        profile = Profile.from_dict(data=data)
+        profile = Profile(**data)
```

Comment:
if data isn't used anywhere below this in the test case, I suggest you make it real arguments of the Profile() instantiation instead

Reply:
You are right. But these tests have become useless; if I do as you rightly suggest, the test becomes identical with the preceding test. So I'll remove them.